app/mailers/user_notifier/mailer.rb

```diff
@@ -21,6 +21,7 @@ def mail_attributes
       from: address_format(UserNotifier.system_email, @notification.from_name),
       reply_to: address_format(@notification.from_email, @notification.from_name),
       to: @notification.user.email,
+      cc: (JSON.parse @notification.metadata.gsub('=>', ':'))['cc'],#FIXME when upgrading to rails 4.2 which supports jsonb
```

Comment:
Is metadata really the place for CC ? For me it seems like standard email notification field, should be on the table.

Any extra field or metadata should be on the migration template: https://github.com/diogob/user_notifier/blob/master/lib/generators/user_notifier/notification/templates/migration.rb

Also could you send a test case in https://github.com/diogob/user_notifier/blob/master/spec/mailers/user_notifier/mailer_spec.rb?